ios-trivia-game/ViewControllers/AnswerViewController.swift

```diff
@@ -43,6 +45,13 @@ class AnswerViewController: UIViewController {
                 Logger.instance.log(logLevel: .info, message: "Adding \(answer.getJson())")
             }
             
+            // add the correct answer
+            let correctAnswer = self.question?.answer ?? ""
+            if (correctAnswer != "") {
+                let answer = Answer(answerText: correctAnswer)
```

Comment:
I think you need to rethink the code here.

In the questionviewcontroller, before you segue to AnswerViewController, you should register the correct answer to the database.

You already have the roomid and the question id as well as the correct answer text.